examples/corrade/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2015, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-corrade)
+
+set(_CORRADE_LIBRARY_COMPONENTS Containers Interconnect PluginManager TestSuite Utility)
```

Comment:
It's up to you, but I personally don't see the point of "yelling" here and of starting variable name with `_`.

It's a user side, I think it will look nicer if you just do:
```cmake
set(components Containers Interconnect PluginManager TestSuite Utility)
```